README.md

```
@@ -180,15 +223,17 @@ export default defineNuxtConfig({
 </template>
 ```
 
-<br />
+## Astro
+
+> :bulb: Import from **`notivue/astro`** (not `notivue`). CSS: **`notivue/astro/notification.css`** and **`notivue/astro/animations.css`** when you use the built-in UI and default animations.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)